src/main/scala/org/broadinstitute/dsde/workbench/sam/Boot.scala

```diff
@@ -336,18 +337,22 @@ object Boot extends IOApp with LazyLogging {
     val managedGroupService =
       new ManagedGroupService(resourceService, policyEvaluatorService, resourceTypeMap, accessPolicyDAO, directoryDAO, cloudExtensionsInitializer.cloudExtensions, config.emailDomain)
     val samApplication = SamApplication(userService, resourceService, statusService, tosService)
-
+    val azureRoutes = config.azureServicesConfig.map { config =>
```

Comment:
Note the `azureServicesConfig` is optional. We won't add the AzureRoutes if it's not present.

This is NOT using the existing `CloudExtensionRoutes` pattern in Sam. `CloudExtensionRoutes` doesn't really fit this use case -- Sam code is structured to only have 1 cloud extension, and it mostly deals with Google group stuff which is not relevant on Azure.